yt/yt/server/master/chunk_server/chunk_replicator.cpp

```diff
@@ -1727,22 +1729,29 @@ void TChunkReplicator::ScheduleJobs(EJobType jobType, IJobSchedulingContext* con
 
 void TChunkReplicator::ScheduleReplicationJobs(IJobSchedulingContext* context)
 {
+    // Keep the ability to disable scheduling completely.
+    if (GetDynamicConfig()->MaxMisscheduledReplicationJobsPerHeartbeat == 0) {
```

Comment:
This is a rather obscure way of doing this, but let's leave it as legacy for now. We can always add more dynconfig parameters later.